editor/src/components/navigator/left-pane/github-pane/index.tsx

```diff
@@ -64,6 +66,10 @@ const GitBranchIcon = () => {
   )
 }
 
+const compactTimeagoFormatter = (value: number, unit: string) => {
+  return `${value}${unit.charAt(0)}`
```

Comment:
It probably doesn't matter, but months and minutes will both give an 'm' here

Reply:
I _think_ it's fine, it's not gonna stick around much and in general, even if you have a wrong indicator it will go back to `s` pretty quickly